src/locales/en/pages.yml

```diff
@@ -1,5 +1,26 @@
-intro:
-  h1: Intro
+answer-security-questions:
+  title: Answer some security questions
+  h1: Answer some security questions
```

Comment:
We'd expect these to always be the same so could we just have `title` and use it in the heading too?

Reply:
As discussed that should work, but is broken due to the frontend using i18next instead of hmpo-i18n